misc-tools/import-contest.in

```diff
@@ -114,6 +114,24 @@ def import_contest_banner(cid: str):
         else:
             print('Skipping contest banner import.')
 
+def import_contest_text(cid: str):
+    """Import the contest text"""
+
+    files = ['contest.pdf', 'contest-web.pdf']
```

Comment:
consider allowing also `.html` - ISTR we also allow it for individual problem texts